rt/rs/microprofile-client/src/main/java/org/apache/cxf/microprofile/client/MicroProfileClientFactoryBean.java

```diff
@@ -97,13 +98,16 @@ protected void initClient(AbstractClient client, Endpoint ep, boolean addHeaders
     @Override
     protected ClientProxyImpl createClientProxy(ClassResourceInfo cri, boolean isRoot,
                                                 ClientState actualState, Object[] varValues) {
+        CDIInterceptorWrapper interceptorWrapper = CDIInterceptorWrapper.createWrapper(getServiceClass());
+        MicroProfileClientProxyImpl impl;
         if (actualState == null) {
-            return new MicroProfileClientProxyImpl(URI.create(getAddress()), proxyLoader, cri, isRoot,
-                    inheritHeaders, executorService, configuration, varValues);
+            impl = new MicroProfileClientProxyImpl(URI.create(getAddress()), proxyLoader, cri, isRoot,
```

Comment:
I think the variant with `return` statements was more readable and straightforward, just an opinion

Reply:
Sure, I can switch it back - I think I changed it for some debugging and just left it, but it's fine with me either way.